highfidelity/hifi-content
# Laser pointer

TEST PLAN
- Run spawnLaser.js. A laser should appear in front of your avatar
- Create a block, only changing the color property to black
- Equip the laser in either hand
- Turn the laser on using the trigger of the hand equipping the laser. The tip of the laser should light up and if there is an item with a collision hull in it's path, a red dot will light up on the intersected item.
- Aim the laser at the block. The red dot should appear on the block and should move as you move the laser pointer in your hand.
- Turn the laser off using the trigger of the hand equipping the laser. The tip of the laser should no longer be lit up and if there is an item with a collision hull in it's path, the red dot will disappear.
- Turn the laser on again and unequip it. The laser should automatically turn off when released.
